app/models/badges/die_unendliche_geschichte_badge.rb

```diff
@@ -0,0 +1,19 @@
+module Badges
+  class DieUnendlicheGeschichteBadge < Badge
+    seed "Die Unendliche Geschichte",
+      :rare,
```

Comment:
1184 users will get this badge. Is that rare or ultimate? Feels like it is rare somehow, as it would be somewhat easy to get this badge if so desired.

Reply:
`rare`